### ISSUE TITLE
Issues following README.md instruction

### ISSUE THREAD

**Issue by Yu-Chieh-Henry-Yang** [DMPRoadmap/roadmap#1580]
**Expected behaviour:**
Following the README.md file instructions should allow me to setup the app without errors/issues

**Actual behaviour:**
1) Some files seem to be missing:
```
$ cp config/locales/*.static.yml.example config/locales/*.static.yml
zsh: no matches found: config/locales/*.static.yml.example
```

2) The README.md seems to forget to include this instruction `cp config/branding_example.yml config/branding.yml` because when I execute `rake secret`, I got:
```
rake aborted!
Could not load configuration. No such file - /Users/my-user-name/dev/roadmap/config/branding.yml
```

3) I use mysql, and running `rake db:schema:load` gives
```
rake aborted!
ActiveRecord::StatementInvalid: Mysql2::Error: Cannot delete or update a parent row: a foreign key constraint fails: DROP TABLE `answers` CASCADE
```
and
```
Caused by:
Mysql2::Error: Cannot delete or update a parent row: a foreign key constraint fails
```
But the workaround is here: https://github.com/DMPRoadmap/roadmap/issues/1569
Basically just remove the default for link

4) Running `rails server` and access http://localhost:3000/ will render a page without images. I got this type of error in `rails server` log:
```
Started GET "/images/dcc_logo_white.png" for ::1 at 2018-06-06 10:08:21 +0930
ActionController::RoutingError (No route matches [GET] "/images/dcc_logo_white.png"):
```
I notice the images asset is stored in lib folder instead of app folder. Why is this? Is this done to help us customise our own roadmap?

**Steps to reproduce:**
Follow instruction on https://github.com/DMPRoadmap/roadmap
Notes: I use mysql

**Comment by Yu-Chieh-Henry-Yang** [comment]
For number 5, the solution is to read /lib/assets/README.md and follow the instruction there. It would be nice if the README.md at https://github.com/DMPRoadmap/roadmap indicate this.

**Comment by briri** [comment]
Thanks for reporting the inconsistencies here and the outdated instructions. 

https://github.com/DMPRoadmap/roadmap/wiki/Installation

The Installation page on the wiki contains all of the correct installation instructions (as far as I know :) ) including both rails and npm. We should probably just update the Readme to point to the wiki page so that we're only maintaining the one set of instructions.

@Yu-Chieh-Henry-Yang for issue 3, it sounds like you have an existing database with data in it. If this is just seed data from a prior installation attempt, you should run `rake db:drop && rake db:create && rake db:schema:load`. If this is your production data, you should NOT run `schema:load` as it will attempt to drop tables. You should instead run `rake db:migrate`. If you are migrating from the older DMPOnline codebase (non-DMPRoadmap) then you should follow the instructions to migrate your data here: https://github.com/DMPRoadmap/roadmap/wiki/Upgrading-from-DMPonline_v4

**Comment by Bodacious** [comment]
hey @briri, can this issue be closed now?

**Comment by xsrust** [comment]
@briri @Bodacious We should double check that the readme is up to date with the release and then close this issue

**Comment by magdalenadrafiova** [comment]
@briri @xsrust is this ok to close?

**Comment by briri** [comment]
yes. we will be updating the readme for Rails5 as part of #2497
